src/test/java/LoginPage/LoginPage.java

```diff
@@ -0,0 +1,42 @@
+package LoginPage;
+
+import ScansPage.ScansPage;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+
+import java.time.Duration;
+
+
+public class LoginPage {
+
+    private final WebDriver driver;
+    public LoginPage(WebDriver driver) {
+        this.driver = driver;
+    }
+
+    public void open() {
+        driver.get("https://app.neuralegion.com/login");
+        driver.manage().timeouts().implicitlyWait(Duration.ofMillis(3000));
+    }
+
+    private final By emailField = By.id("mat-input-0");
+    private final By passwordField = By.id("mat-input-1");
```

Comment:
Let's use `*Locator` name convention for locators. This will help us when we'll want to retrieve some elements on the page to check if it is disabled, has some class etc. ex:
```java
class LoginPage {
    private final By signInBtnLocator = By.cssSelector("auth-login-form form > button");

    public WebElement getSignInButton() {
        return driver.findElement(signInBtnLocator);
    }
}

// in test

class LoginPageTest {
    public checkIsSignInBtnDisabled() {
        LoginPage page = new LoginPage();
        Assert.assertTrue(page.getSignInButton().disabled);
    }
}


```